roles/django_app_docker/defaults/main.yml

```diff
@@ -121,6 +121,15 @@ django_app_docker_celery_db: 1  # uses the same redis instance as cache service
 django_app_docker_broker_url: "redis://{{ django_app_docker_name_prefix }}-redis:6379/{{ django_app_docker_celery_db }}"
 django_app_docker_result_backend: "{{ django_app_docker_broker_url }}"
 
+# RabbitMQ
+django_app_docker_use_rabbitmq: false
+# django_app_docker_broker_url: "amqp://{{ django_app_docker_rabbitmq_user }}:{{ django_app_docker_rabbitmq_pass }}@{{ django_app_docker_name_prefix }}-rabbitmq:5672//"
+# django_app_docker_result_backend: "redis://{{ django_app_docker_name_prefix }}-redis:6379/{{ django_app_docker_celery_db }}"
+django_app_docker_publish_broker_url: "amqp://{{ django_app_docker_rabbitmq_user }}:{{ django_app_docker_rabbitmq_pass }}@{{ django_app_docker_name_prefix }}-rabbitmq:5672/%2F"
```

Comment:
`PUBLISH_BROKER_URL` is not a standard django thing, but Open Notificaties specific. It's not even actively used in there, so I think if rabbitmq is enabled, we can just set `django_app_docker_broker_url` to the rabbitmq host, as this is afaik the only context (celery message broker)

For Open Notificaties specifically, we can specify an extra custom envvar and build it up ourselves.

Reply:
Oke removed.